commands/database-get.js

```diff
@@ -73,34 +63,14 @@ module.exports = new Command('database:get <path>')
 
       url += querystring.stringify(query);
 
-      request.get(url)
-        .on('response', function(res) {
-          response = res;
-          if (response.statusCode >= 400) {
-            erroring = true;
-          }
-        })
-        .on('data', function(chunk) {
-          if (erroring) {
-            errorResponse += chunk;
-          } else {
-            outStream.write(chunk);
-          }
-        })
-        .on('end', function() {
-          if (erroring) {
-            try {
-              var data = JSON.parse(errorResponse);
-              return reject(responseToError(response, data));
-            } catch (e) {
-              return reject(new FirebaseError('Malformed JSON response', {
-                exit: 2,
-                original: e
-              }));
-            }
-          }
-          return resolve();
-        })
-        .on('error', reject);
+      api.request('GET', url, {
```

Comment:
Some background on why this looked this way: I specifically designed database requests to stream data from the raw response since you might be fetching a huge chunk of data. By swapping to the generic `api.request` we are going to do two bad things:

1. We won't print anything out until the entire request is returned
2. The entire response body will be parsed into memory, which could be enormous and costly

I think in this case it might make more sense to keep the raw request nearly as-is and simply create or utilize a helper method to get the access token for use in the header. Make sense?